src/commands/build.ts

```diff
@@ -49,6 +60,17 @@ export default class Build extends BaseCommand {
       description:
         "Switch between production more or development mode (with additional validation generated for some languages)",
     }),
+    token: Flags.string({
+      char: "t",
+      required: false,
+      description: "Pass API token for fetching the tracking plan from Segment",
+    }),
+    path: Flags.boolean({
+      char: "p",
+      required: false,
+      default: false,
+      description: "Update path for tracking plan files",
+    }),
```

Comment:
I think this is fine but it doesn't fully fix the original issue of trying to not have any configuration files. How about we make all these `WorkspaceConfig` values possible flags:

```
  client: {
    language: string;
    sdk: string;
  } & T;
  /** Which Tracking Plans to sync locally and generate clients for. */
  trackingPlans: TrackingPlanConfig[];
```

(We don't need the scripts part)

We can have the flag arguments be something like:
`typewriter -t "TOKEN" -i "TrackingPlanId" -p "path" -l language -s sdk`

We can limit this way of calling TW to a single plan for now. 

We don't need the `name` iirc, we only use it for legacy stuff.

Eventually we can add support for multiple plans given that OCLIF does have the ability for multiple flags, but for now I think it's fine that if people want to use it this way they call it for each TP.